validator/sawtooth_validator/journal/batch_injector.py

```diff
@@ -50,6 +50,48 @@ def block_start(self, previous_block):
         """
         raise NotImplementedError()
 
+    @abc.abstractmethod
+    def before_batch(self, previous_block, batch):
+        """Returns an ordered list of batches to inject at the beginning of the
```

Comment:
This comment does not match the method. (same with the other two added methods)

Reply:
Thank you for your time, I'll fix these typos that you pointed out as soon as possible